app/routes/wiki-page.js

```diff
@@ -190,6 +191,12 @@ export default Route.extend(
                 { useskin: 'oasis' },
               ),
             });
+          } else {
```

Comment:
I think setting surrogate key should be possible regardless of the redirect, so this should be moved one level up, before https://github.com/Wikia/mobile-wiki/pull/1492/files#diff-d77b3e7a178bee9c8affd145b2261a54R184.
Second, I think you should wrap it with the `if (fastboot.get('isFastBoot')) {` check